pkg/cmd/run.go

```diff
@@ -40,9 +40,6 @@ func NewRunCommand() *cobra.Command {
 }
 
 func run(_ *cobra.Command, _ []string) {
-	ctx, stop := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
```

Comment:
This needed to move after the initialization of the database. Otherwise, I was unable to ctrl-c out of the "waiting for Postgres" loop until the backoff expired.